lua/lspconfig/denols.lua

```diff
@@ -103,7 +103,14 @@ configs[server_name] = {
   default_config = {
     cmd = { 'deno', 'lsp' },
     filetypes = { 'javascript', 'javascriptreact', 'javascript.jsx', 'typescript', 'typescriptreact', 'typescript.tsx' },
-    root_dir = util.root_pattern('package.json', 'tsconfig.json', '.git'),
+    root_dir = function(fname)
+      local root_files = {
+        'package.json',
+        'tsconfig.json',
+        '.git',
```

Comment:
This `.git` is redundant since you have `find_git_ancestor` below.

Reply:
Thank you for your suggestion.